platooning_strategic_IHP/include/platoon_strategic_plugin_node_ihp.h

```diff
@@ -77,10 +77,10 @@ class PlatoonStrategicIHPPluginNode
     pnh.param<double>("desiredJoinGap", config.desiredJoinGap, config.desiredJoinGap);
     pnh.param<double>("waitingStateTimeout", config.waitingStateTimeout, config.waitingStateTimeout);
     pnh.param<double>("cmdSpeedMaxAdjustment", config.cmdSpeedMaxAdjustment, config.cmdSpeedMaxAdjustment);
-    pnh.param<double>("minAllowableHeadaway", config.minAllowableHeadaway, config.minAllowableHeadaway);
-    pnh.param<double>("maxAllowableHeadaway", config.maxAllowableHeadaway, config.maxAllowableHeadaway);
-    pnh.param<double>("headawayStableLowerBond", config.headawayStableLowerBond, config.headawayStableLowerBond);
-    pnh.param<double>("headawayStableUpperBond", config.headawayStableUpperBond, config.headawayStableUpperBond);
+    pnh.param<double>("lowerBoundary", config.lowerBoundary, config.lowerBoundary);
```

Comment:
It looks like all the changes here are obsolete?  The names don't match anything in the platoon_config_ihp.h.  Not sure how it compiles.  In any case, names like "lowerBoundary" are too generic - there could be many physical quantities that have a lower boundary, so something more descriptive is needed, especially for a parameter file that needs to stand alone.

Reply:
Bad version control. Updated and recompiled.